frontend/packages/dev-console/src/components/pipelineruns/PipelineTaskStatus.tsx

```diff
@@ -0,0 +1,87 @@
+/* eslint-disable no-undef */
+import * as React from 'react';
+import { VictoryStack, VictoryBar } from 'victory';
+import { Tooltip } from '@console/internal/components/utils/tooltip';
+import { getTaskStatus, PipelineRun } from '../../utils/pipeline-augment';
+import '../pipelines/PipelineVisualizationTask.scss';
+
+interface PipelineTaskStatusProps {
+  pipelinerun: PipelineRun;
+}
+
+interface TaskStatusToolTipProps {
+  tooltip: Tooltip;
+}
+
+interface Tooltip {
+  Succeeded?: number;
+  Running?: number;
+  Failed?: number;
+  Notstarted?: number;
+  FailedToStart?: number;
+}
+
+//The colors are not final. pf-colors are not supported in the charts
+export const mapColorMessage = {
+  Succeeded: { color: 'green', message: 'Succeded' },
+  Running: { color: '#0066cc', message: 'Running' },
+  Failed: { color: 'red', message: 'Failed' },
+  Notstarted: { color: '#ccc', message: 'Not Started Yet' },
+  FailedToStart: { color: 'red', message: 'PipelineRun Failed To Start' },
+};
+
+const getMessage = (status: string, value: number = 0): string => {
+  if (status === 'Notstarted' || status === 'FailedToStart') {
+    return mapColorMessage[status].message;
+  }
+  return `${value} ${mapColorMessage[status].message}`;
+};
+
+export const TaskStatusToolTip: React.FC<TaskStatusToolTipProps> = (props) => {
+  const { tooltip } = props;
+  return (
+    <div className="odc__task__status">
+      {Object.keys(tooltip).map((status) => {
+        return (
+          <div className="row" key={status}>
+            <div className="col-lg-1 col-md-1 offset-lg-1 offset-md-1">
```

Comment:
Use a fixed padding to indent the content instead of using offsets.
Shouldn't you just use `col-*` so that there is never any stacking?